_posts/2017-02-07-women-technology-careers.md

```diff
@@ -0,0 +1,22 @@
+---
+layout:     post
+title:      How do we increase the numbers of women in technology based careers?
+date:       2017-02-07 18:00
+summary:    How to tackle the technology industry’s disheartening gender imbalance from the perspective of a female software engineer.
+image:      women-in-it-tickets.jpg
+category:   Community
+tags:       women, career
+author:     sophia_chambers
+---
+
+The structure of social scales begins at Primary school, you begin to develop your own acceptance criteria of what is socially acceptable. This usually is a structure which is designed and implemented by your group of social peers. Other variables are contributed as well, for example: individual likes and dislikes, tendencies, personality traits all these combined will create your visual implementation of your social hierarchy.
+
+So, what does a social acceptance criteria have to do with being a woman in technology? The answer – a hell of a lot.
+In my mass research with companies who are exploring advertising campaigns to reach interest of women to improve their diversity levels, they all seem to miss one thing. Prevention is always better than the cure. In this sense, this regards dynamics and age. The usual target-audience for women in tech campaigns by companies, is usually graduation age, due to most technical companies holding vacancies for graduates from university. This will not solve the issue. Many technology companies will take on any degree (like many grad schemes) and put them on an intensive graduate programme to get their training up to scratch sufficiently to do their required future job role. Therefore, imagine doing a degree in psychology and applying for a graduation scheme within a software company for coding, would it be an event you would even attend out of interest? However, the psychologist graduate may not be aware that, for example, UX design, which is a role developed to understand the psychology of our users and their experience is highly related to software, thus brings in it’s own opportunities. Women in particular seem to be completely ‘bored’ or misunderstand tech, why? Simple, it is not usually a socially acceptable role.
```

Comment:
...advertising campaigns to reach **interest of women** to improve...

should that be "interested women"?